### ISSUE TITLE
Character encoding checks trigger on 'Wrong character encoding (_ instead of "")'

### ISSUE THREAD

**Issue by CasperRademaker-AbOvo** [editorconfig-checker/editorconfig-checker#505]
Great tool!

Since release 3.5.0 which seems to include file character encodings checks, I'm getting a few findings that are seemingly false positives:

```
> ec .

.editorconfig:
	Wrong character encoding ("Ascii" instead of "")
readme.md:
	Wrong character encoding ("UTF-8" instead of "")
```

my `.editorconfig`:

```
[*]
insert_final_newline = true

[*.{js,json,json5,ts,ps1,py,sql,tsql,yaml,yml}]
charset = utf-8
indent_style = space
trim_trailing_whitespace = true

[*.{md}]
charset = utf-8
indent_style = space

[*.py]
indent_size = 4

[*.{js,ts}]
indent_size = 2
```

The detected encodings seem correct (ascii and UTF-8), but it is compared against an undefined expected encoding.

**Comment by iloabn** [comment]
@theoludwig can that also be applied in the action in the same way?

Aka, would a workflow like this disable that feature?

```yaml
      - uses: editorconfig-checker/action-editorconfig-checker@main
      - run: editorconfig-checker -disable-charset
```
